File: Sigmath/Sigmath/Sigmath.ConvertedToVBNet/engineering/mechanical.vb
Class Mechanical
	Public Shared Function CalculateTorque(force As Double, radius As Double) As Double
		Return force * radius
	End Function

	Public Shared Function CalculatePower(torque As Double, angularVelocity As Double) As Double
		Return torque * angularVelocity
	End Function

	Public Shared Function CalculateVelocity(distance As Double, time As Double) As Double
		Return distance / time
	End Function

	Public Shared Function CalculateAcceleration(initialVelocity As Double, finalVelocity As Double, time As Double) As Double
		Return (finalVelocity - initialVelocity) / time
	End Function

	Public Shared Function CalculateForce(mass As Double, acceleration As Double) As Double
		Return mass * acceleration
	End Function
End Class

File: Sigmath/Sigmath/Sigmath.ConvertedToRuby/engineering/mechanical.rb
module Sigmath
	class Mechanical
		def Mechanical.CalculateTorque(force, radius)
			return force * radius
		end

		def Mechanical.CalculatePower(torque, angularVelocity)
			return torque * angularVelocity
		end

		def Mechanical.CalculateVelocity(distance, time)
			return distance / time
		end

		def Mechanical.CalculateAcceleration(initialVelocity, finalVelocity, time)
			return (finalVelocity - initialVelocity) / time
		end

		def Mechanical.CalculateForce(mass, acceleration)
			return mass * acceleration
		end
	end
end